client/src/styles/_common.scss

```diff
@@ -49,17 +49,60 @@ code {
 .statistics-card {
   border: none;
   background-color: $primary-color;
+  border-radius: 16px;
+  padding: 30px 10px 10px;
 
   &, & h3, & h4 {
     color: #fff;
   }
 
   & h3, & h4 {
     text-align: left;
+    margin-bottom: 0;
   }
 
-  & svg {
-    width: 38px;
-    height: 38px;
+  & .stats {
```

Comment:
I think it might be easier to follow if we changed the name to `stats-value`

Reply:
Changed name to `stats-heading`.